docs/docs/guides/airy-core-in-production.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Running the Airy Core Plaform in production
+sidebar_label: Production
+---
+
+This document provides an overview of the architecture of the Airy Core Platform and good practices for production environments.
+
+## Architecture
+
+The Airy Core Platform is a messaging platform comprised of a backend and frontend system, both of which run on top of multiple docker containers.
+
+The `backend` system is a streaming platform and its role is to:
+- Receive messages or events from different sources, process them on the fly and store them in Kafka [sources](glossary.md#sources)
+- Make them available and accessible through an [API](api/http.md)
+- Send messages or events to external parties [webhooks](api/webhook.md)
+
+Furthermore, the platform provides an `admin` application which manages authentication and authorization features.
+
+The `frontend` system is comprised of a demo application and the JavaScript integration of  the [Chat Plugin](sources/chat-plugin.md).
+
+Having that in mind, the docker containers or `Airy apps` which need to run in order for the Airy Core Platform to be fully functional, have the following naming convention:
+
+### Sources
+- sources-`SOURCE_NAME`-webhook - Receive messages from a specific source
+- sources-`SOURCE_NAME`-events-router - Process messages from a specific source
+- sources-`SOURCE_NAME`-sender - Send messages to a specific source
+
+### API
+- api-`ENTITY` - Backend services, exposed to the outside (admin, authentication, communication, etc.)
+
+### Webhook
+- webhook-`ENTITY` - Process events to be sent to external parties
+
+### Frontend
+- frontend-`ENTITY` - Frontend application, communicating with the `API` services
+
+
+## Requirements 
+
+The `Airy apps` should be started as separated docker containers and they must have access to several other underlying services, which need to be in place for the system to be fully functional:
+- `Kafka cluster`: Kafka, Zookeeper and the Schema registry. These three services comprise the kafka store and need to be in place as they are the base store for messages and events which are received and processed by the Airy Core Platform. Kafka needs Zookeeper to be in place to work and the Schema registry is an application which is the central schema repository for all the data written in kafka, used by the streaming apps inside the Airy Core Platform.
+- `PostgreSQL`: Holding a database which stores organization and user related data.
+- `Redis`: Data structure used as a simple message queue, for events which are sent to external webhooks.
+
+Apart from the underlying services, a docker runtime or a running Kubernetes cluster must also be available, for deploying the `Airy apps`.
```

Comment:
```suggestion
A docker runtime or a running Kubernetes cluster must also be available to deploy the `Airy apps`.
```

Reply:
right in the next paragraph we say it can be run anywhere... so this comes across as confusing. I think it's ok we require Kubernetes for now (it's also simpler to deploy so many services right now this way)

Reply:
On `any` infrastructure, so I think that is still accurate. One can run all of  the base services and kubernets or docker engine on his own infrastructure.

But maybe instead of  on `any infrastructure` I can put `on your own infrastructure`? But `any` also indicates that it can be run in your data center or your custom hosting provider...